js/base/Exchange.js

```diff
@@ -1132,6 +1158,24 @@ module.exports = class Exchange {
         return indexed ? indexBy (result, key) : result
     }
 
+    sortArrayObjects (unsortedArray, sortDictKey) {
```

Comment:
We have `this.sortBy` in the base class, also, `this.groupBy` and `this.indexBy`. What's the purpose of `sortArrayObjects` here? What should it do? Asking this, because I think it could be done without adding a new base method, depends on the task in question. Can you elaborate on that a bit, plz?

Reply:
Thanks again for pointing this out, didn't realize these functions existed. The purpose was to ensure the Array of Objects was sorted correctly by timestamp, i.e. as the docs indicate most recent last in the array. Have removed from the base method.

Reply:
Awesome! thx to you as well for your involvement! I will review this once again asap and will let you know if anything left.